pkgs/tools/X11/xosview2/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, libX11 }:
+
+stdenv.mkDerivation rec {
+  name = "xosview2-${version}";
+  version = "2.2.2";
+
+  src = fetchurl {
+    url = "mirror://sourceforge.net/xosview/${name}.tar.gz";
```

Comment:
This is wrong. Should be without ".net":
```
url = "mirror://sourceforge/xosview/${name}.tar.gz";
```

It's good to merge after this is fixed.